test/test_core.py

```diff
@@ -9418,14 +9418,13 @@ def test_Module_dynamicLibraries_pthreads(self):
     self.set_setting('NO_AUTOLOAD_DYLIBS')
 
     create_file('pre.js', '''
-      if (!global.Module) {
-        // This is the initial load (not a worker)
-        // Define the initial state of Module as we would
-        // in the html shell file.
-        // Use var to escape the scope of the if statement
-        var Module = {
-          dynamicLibraries: ['liblib.so']
-        };
+      const sideModule = 'liblib.so';
+      if (typeof importScripts == 'undefined') { // !ENVIRONMENT_IS_WORKER
+        // Load liblib.so by default on non-workers
+        Module['dynamicLibraries'] = [sideModule];
```

Comment:
I think for tests its maybe simpler/clearer if you just duplicate the 'liblib.so' string?  but maybe thats just personal preference.

Reply:
1ce91e17dee1b30f3540ca3c732fd6c1205a3d05

Reply:
Let me know if I should polish the commit message too.